### ISSUE TITLE
Question on use with MAGs

### ISSUE THREAD

**Issue by AstrobioMike** [AlexanderLabWHOI/EUKulele#23]
Hi there!

Thanks for all the hard work and documentation!

I might be just confusing myself somehow or overlooking something, but it seems that here on the quick start page (https://eukulele.readthedocs.io/en/latest/quickstart.html#metagenome-assembled-genome-mag-annotation), it talks about using input nucleotide fastas for MAGs and doesn't mention that being a problem. But then on the using eukulele page here (https://eukulele.readthedocs.io/en/latest/running-eukulele.html#metagenome-assembled-genomes-mags) it says we shouldn't use nucleotide contigs, and should predict proteins first. 

Am I overlooking something here, or maybe you folks learned from use/testing it's not a good idea to use on MAG contigs and just haven't been able to update the docs everywhere yet? 

Thanks!
-Mike

**Comment by akrinos** [comment]
Hi! Thanks so much for checking out our tool!

We indeed recommend that protein prediction be done before using `EUKulele`. For METs, we provide the option to use `blastx` on nucleotide files rather than `blastp` (using either BLAST or DIAMOND), but for MAGs we recommend that the protein prediction process occur outside of `EUKulele`, as the only protein prediction tool that we have built into the software is `TransDecoder`. 

Of course, always open to feedback!

Arianna
